dpctl/_sycl_device_factory.pyx

```diff
@@ -162,7 +162,7 @@ cpdef list get_devices(backend=backend_type.all, device_type=device_type_t.all):
         device_type (optional):
```

Comment:
It seems the docstring of `device_type` needs to updated also, like [here](https://github.com/IntelPython/dpctl/pull/1992/files#diff-83d5f19556e2bf2011ee51924c99979156bacd405cfb4351bfbb39bb661a42bcR385)
```suggestion
        device_type (str, :class:`dpctl.device_type`, optional):
```